src/main/java/org/ibp/api/rest/location/LocationResource.java

```diff
@@ -51,13 +51,22 @@ public ResponseEntity<LocationDTO> getLocation(
 		return new ResponseEntity<>(this.locationService.getLocation(locationId), HttpStatus.OK);
 	}
 
-	@ApiOperation(value = "Get default location")
-	@RequestMapping(value = "/crops/{cropName}/programs/{programUUID}/locations/default", method = RequestMethod.GET)
+	@ApiOperation(value = "Get breeding location default")
+	@RequestMapping(value = "/crops/{cropName}/programs/{programUUID}/locations/breeding-location-default", method = RequestMethod.GET)
```

Comment:
If I have to implement it I will leave as it was (/default) and send a type by param to the GET function so we have only one default service per program 
@mderamos